include/pfs/types.hpp

```diff
@@ -604,6 +604,22 @@ struct id_map
     size_t length = UINT32_MAX;
 };
 
+// Based on https://elixir.bootlin.com/linux/v5.19.10/source/net/ipv4/fib_trie.c#L2998
+struct route
+{
+    std::string iface;
+    ip destination;
+    ip gateway;
+    unsigned int flags;
+    unsigned short refcnt;
+    unsigned short use;
+    uint32_t metric;
+    ip mask;
+    uint32_t mtu;
```

Comment:
Same comment for `mtu`, `window` and `irtt` as above. We should be consistent with the `printf` formatter? Being consistent with the type might create discrepancies, being consistent with the formatter cannot.